npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/version_resolver.rb

```diff
@@ -302,26 +304,34 @@ def updated_types_dependencies
         end
 
         def peer_dependency_errors
-          return @peer_dependency_errors if @peer_dependency_errors_checked
+          resolution_errors.reject { |error| error == "NOT_RESOLVABLE" }
```

Comment:
Where does this `"NOT_RESOLVABLE"` come from? pnpm itself, or something within Dependabot?

Reply:
I removed this commit. It was an improvement for an edge case (trying to update to a version with yanked deps), but it's not specific to PNPM, so I'll propose it separately if needed.